airbyte-integrations/connectors/source-salesforce/source_salesforce/spec.json

```diff
@@ -60,24 +66,49 @@
                 "not contains",
                 "not exacts"
               ],
-              "default": "contains"
+              "default": "contains",
+              "order": 201
             },
             "value": {
               "type": "string",
-              "title": "Search value"
+              "title": "Search value",
+              "order": 202
             }
           }
         },
         "title": "Streams filter criteria",
-        "description": "Add selection criteria for streams to get only streams that are relevant to you"
+        "description": "Add selection criteria for streams to get only streams that are relevant to you",
+        "order": 110
+      },
+      "exclude_fields": {
+        "title": "Fields to Exclude",
+        "description": "Add any fields that should always be excluded using Stream.Field syntax",
+        "examples": ["Contact.SSN", "CustomerPayment.PrimaryCC"],
+        "type": "array",
+        "items": {
+          "type": "string",
+          "pattern": "^[A-Za-z0-9_]+\\.[A-Za-z0-9_]+$"
```

Comment:
@marcosmarxm updated the spec here to enforce the Stream.Field syntax - would be best to catch these issues up front!